tac/project-updates/2025/2025-annual-Besu.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: 2024 Annual Review Besu
+parent: Project Updates
+grand_parent: LF Decentralized Trust TAC
+nav_exclude: true
+---
+
+
+
+## Project Health
+
+The Besu project continues to thrive. [Fleet Mode](https://consensys.io/blog/besu-fleet-the-future-of-rpc-scaling) has bolstered interest in Besu from public blockchain networks and private chains, with both Layer 2 and RPC providers considering Besu for their infrastructure. Community activity on Discord is stable, though external contributions are at a seasonal lull. Overall, the project remains healthy.
+
+
+## Maintainer Diversity
+
+The Besu project currently has 21 maintainers, 19  of which work for Consensys Software Inc. The remaining 2 work for Kaleido. Existing maintainers give special attention to new pull requests and are very timely in moving those forward.
+
+## Project Adoption
+
+Besu's share of mainnet stake is around 18%. This value is very hard to measure reliably, but anywhere between 15-20% is a good place for an execution client to be.
+
```

Comment:
It's probably worth including a reference to the adoption of Besu in the enterprise space, where it's probably 50%+ of blockchain clients, and 90%+ of Ethereum clients.